gen/arpresolv_linux.c

```diff
@@ -88,6 +89,12 @@ struct recvnd_arg {
 static unsigned char buf[BUFSIZE];
 static unsigned int buflen = BUFSIZE;
 
+#ifdef DEBUG
+#define __diagused
```

Comment:
Ditto. In this case `__debugused` is appropriate.

Reply:
I'll do it.

Reply:
done